wagtail/blocks/stream_block.py

```diff
@@ -766,6 +772,11 @@ def __reduce__(self):
         )
 
 
+# Now that StreamValue is defined, make it the default `value_class`
+# for all `BaseStreamBlock` subclasses
+BaseStreamBlock.default_value_class = StreamValue
```

Comment:
Since StreamValue's implementation doesn't contain any direct reference to StreamBlock, I'd be happier if we moved the StreamValue class above StreamBlock to avoid having to do this dance (and match what happens in struct_block.py).